tests/tracing/test_splining.py

```diff
@@ -21,6 +23,65 @@
 # pylint: disable=too-many-arguments
 
 
+def plot_spline_debugging(
+    image: npt.NDArray[np.float32],
+    result_all_splines_data: dict,
+    pixel_to_nm_scaling: float,
+):
+    """Plot splines of an image overlaid on the image.
```

Comment:
Numpydoc would probably complain this isn't on a new line (although we have `^test_` in place which has prevented it being caught here!).

Clearly my brain has been infected though if I notice that.

Reply:
Done, thanks! I didn't notice that those listing steps are ignored on test files